api/include/mfxstructures.h

```diff
@@ -272,7 +272,12 @@ enum {
 
 /* Transcoding Info */
 typedef struct {
-    mfxU32  reserved[7];
+    mfxU32  reserved;
+    mfxU32  MaxIFrameSize;
+    mfxU32  MaxPFrameSize;
+    mfxU32  MaxBFrameSize;
+    mfxU32  NumPasses;
+    mfxU8   DeltaQP[8];
 
```

Comment:
I'm against extending of topmost mfx structure mfxInfoMFX with very custom BRC related parameters. It would be better to create additional ExtBuffer. Custom feature should be isolated as much as possible.

Reply:
Not that problem actually, change is done right way in general, but we already have most of these options in mfxExtCodingOption3 (MaxFrameSize for I and P frames and place for different B frame as well as delta QP)

Reply:
Agree, mfxExtCodingOption3 seems to be the best choice.